docs/source/ui/pages/bookmarks.md

```diff
@@ -2,40 +2,50 @@
 
 ### Overview
 
-Bookmarks page represents [cards](#bookmark-card) for quick access to the explorer from which it was created.
+Bookmarks are used to save a given configuration of an explorer (search settings, aggregations, groupings, etc) for later access. The bookmarks page contains the bookmark [cards](#bookmark-card) for quick access to the explorer from which they were created.
 
 <img alt="bookmarks" style="border-radius: 8px; border: 1px solid #E8F1FC" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/bookmarks/bookmarks.png">
+<p>&nbsp;</p>
 
-Initially page is empty.
+Initially the page is empty.
 
-<img alt="bookmarks" style="border-radius: 8px; border: 1px solid #E8F1FC" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/bookmarks/bookmarks-empty.png">
+<img alt="bookmarks" style="border-radius: 8px; border: 1px solid #E8F1FC" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/bookmarks/bookmarks-empty.png"> 
+<p>&nbsp;</p>
 
-To create bookmark you should open one of our explorer pages and from top bar click `bookmark` button
+To create a bookmark, open one of our explorers and click the `bookmark` button on the top right.
```

Comment:
There are `bookmark` buttons available on all Explorer pages - on the top right .  In order to create a bookmark, just press the `bookmark` button on your preferred Explorer page.